packages/react-playground/src/App.js

```diff
@@ -37,7 +39,17 @@ class App extends React.Component {
     this.setState({ disabled: !this.state.disabled });
   }
 
+  setSize(event) {
+    this.setState({ size: event.target.value });
```

Comment:
Value is a computed property it's:

```state.options[state.selectedIndex]```. You think the state should have the 'value' directly?